src/OrchardCore.Modules/OrchardCore.Admin/Startup.cs

```diff
@@ -54,7 +54,11 @@ public override void ConfigureServices(IServiceCollection services)
             services.AddScoped<IPermissionProvider, PermissionsAdminSettings>();
             services.AddScoped<INavigationProvider, AdminMenu>();
 
-            services.Configure<AdminOptions>(_configuration.GetSection("OrchardCore.Admin"));
+            var adminSection =
+                _configuration.GetSection("OrchardCore_Admin") ??
```

Comment:
A returned section is never null but may have no value and no children

This can be checked by using `section.Exists()`

Reply:
Good catch. I'll double-check this everywhere.